ramani-maplibre/src/main/java/org/ramani/compose/MapLibre.kt

```diff
@@ -96,6 +97,7 @@ fun MapLibre(
     uiSettings: UiSettings = UiSettings(),
     properties: MapProperties = MapProperties(),
     locationRequestProperties: LocationRequestProperties? = null,
+    locationEngine: LocationEngine? = null,
```

Comment:
Do you mind adding it to the docs above? 🙏

Reply:
Sure, I usually do it as well.